src/main/java/emu/grasscutter/command/commands/SetSceneTagCommand.java

```diff
@@ -0,0 +1,98 @@
+package emu.grasscutter.command.commands;
+
+import emu.grasscutter.command.*;
+import emu.grasscutter.data.GameData;
+import emu.grasscutter.data.excels.scene.SceneTagData;
+import emu.grasscutter.game.player.Player;
+import emu.grasscutter.server.packet.send.*;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+
+import java.util.*;
+
+@Command(label = "setSceneTag", aliases = { "tag" }, usage = {
+        "<add|remove|unlockall> <sceneTagId>" }, permission = "player.setscenetag", permissionTargeted = "player.setscenetag.others")
+public final class SetSceneTagCommand implements CommandHandler {
+    private final Int2ObjectMap<SceneTagData> sceneTagData = GameData.getSceneTagDataMap();
+
+    @Override
+    public void execute(Player sender, Player targetPlayer, List<String> args) {
+        if (args.size() == 0) {
+            sendUsageMessage(sender);
+            return;
+        }
+
+        String actionStr = args.get(0).toLowerCase();
+        int value = -1;
```

Comment:
```suggestion
        var actionStr = args.get(0).toLowerCase();
        var value = -1;
```

Reply:
at least actionStr doesn't change, so `val` would be even better for it ^^